merlin/main.py

```diff
@@ -126,16 +132,16 @@ def parse_override_vars(variables_list):
                     f"Cannot override reserved word '{key}'! Reserved words are: {RESERVED}."
                 )
 
-            val = entry[1]
+            val: Union[str, int] = entry[1]
             with suppress(ValueError):
                 int(val)
                 val = int(val)
             result[key] = val
 
-        except BaseException as e:
+        except BaseException as excpt:
```

Comment:
Does excpt need typing?

Reply:
I don't believe so..? It's implicitly a BaseException, so, any Exception at all :D. I suppose it could be forward declared like 
```
excpt: BaseException
except BaseException as excpt:
```
but, I'm not sure if that would work with the context management of the `except` clause, bc I think that the forward declared var wouldn't actually be used. Kind of like forward declaring the temp var used in a list comprehension causes  "unused var" linter problems for the forward declaration.